eisbuk-admin/src/components/atoms/AttendanceCard/__tests__/AttendanceCard.test.tsx

```diff
@@ -54,10 +68,61 @@ describe("AttendanceCard", () => {
     });
     test("should disable attendance button while state and fb are synching", () => {
       screen.getByText("👎").click();
-      expect(screen.getByTestId("SaulGoodman")).toHaveProperty(
+      expect(screen.getByTestId("Saul-Goodman")).toHaveProperty(
         "disabled",
         true
       );
     });
+    /** @TODO find a good test for options */
+    test("should render intervals in dropdown when athlete is present", () => {
+      screen.getByText("👎").click();
+      expect(screen.getByTestId("Saul-Goodman-input")).toBeInTheDocument();
+
+      // expect(screen.getAllByTestId("SaulGoodman13:00 - 14:00").length).toEqual(
+      //   customersSlot.intervals.length
+      // );
+    });
+    test("should render bookedInterval value as display value", () => {
```

Comment:
`"should use bookedInterval as initial interval value"` seems a bit more accurate